src/app/plugins/modreport.plugin.ts

```diff
@@ -70,69 +79,55 @@ export default class ModReportPlugin extends Plugin {
     return { subCommand, givenHandle, description };
   }
 
-  private async _createReport(message: IMessage, user_handle: string, description?: string) {
-    const id = await Moderation.Helpers.resolveToID(this.container.guildService.get(), user_handle);
-
-    if (!id) {
-      return;
-    }
-
-    const rep: Moderation.Report = new Moderation.Report(
+  private _createReport(message: IMessage, id: string, description?: string) {
+    return new Moderation.Report(
       this.container.guildService.get(),
       id,
       description,
       message.attachments.map((e) => e.url)
     );
-
-    return rep;
   }
 
-  private async _handleAddReport(message: IMessage, user_handle: string, description?: string) {
-    const rep = await this._createReport(message, user_handle, description);
+  private async _handleAddReport(message: IMessage, id: string, description?: string) {
+    const rep = this._createReport(message, id, description);
     if (!rep) {
       await message.reply('Error creating report');
       return;
     }
     message.reply(await this.container.modService.fileReport(rep));
   }
 
-  private async _handleListReport(message: IMessage, user_handle: string) {
+  private async _handleListReport(message: IMessage, id: string) {
     this.container.messageService.sendStringOrEmbed(
       message.channel as TextChannel,
-      await this.container.modService.getModerationSummary(
-        this.container.guildService.get(),
-        user_handle
-      )
+      await this.container.modService.getModerationSummary(this.container.guildService.get(), id)
     );
   }
 
-  private async _handleFullList(message: IMessage, user_handle: string) {
+  private async _handleFullList(message: IMessage, id: string) {
     try {
       await message.reply({
-        content: `Full Report for ${user_handle}`,
+        content: `Full Report for ${id}`,
```

Comment:
can we pass both `user_handle` and `id` just for this ??

Reply:
Can fetch the user handle from the ID, I just want all these methods to be standardized